database/init.sql

```diff
@@ -49,7 +49,9 @@ CREATE TABLE manager (
 
 CREATE TABLE appointment_time (
```

Comment:
Should we name this something different? Like branch_hours or appointment_hours?